rust/perspective-python/perspective/handlers/tornado.py

```diff
@@ -29,35 +26,38 @@ class PerspectiveTornadoHandler(PerspectiveHandlerBase, WebSocketHandler):
     alive without timing out.
 
     Examples:
-        >>> MANAGER = PerspectiveManager()
-        >>> MANAGER.host_table("data_source_one", Table(
-        ...     pd.read_csv("superstore.csv")))
+        >>> server = psp.Server()
+        >>> client = server.new_client()
+        >>> client.table(pd.read_csv("superstore.csv"), name="data_source_one")
         >>> app = tornado.web.Application([
         ...     (r"/", MainHandler),
         ...     (r"/websocket", PerspectiveTornadoHandler, {
-        ...         "manager": MANAGER,
+        ...         "perspective_server": server,
         ...         "check_origin": True
         ...     })
         ... ])
     """
 
     def __init__(self, *args, **kwargs):
```

Comment:
no init needed, args passed in via `initialize` for tornado: https://github.com/finos/perspective/pull/2664/files#diff-f6dab3bf154b1234938c04af999f4d052c4d9f6e5fa7220cb789ccf3512e15acR19